docs/rpc_nodes.md

```diff
@@ -15,23 +15,20 @@ author: Roxane Letourneau
 
 *If you are aware of a public node missing from our list or our information is inaccurate, please help us by submitting an issue or pull request on our GitHub page.*
 
-- ECAD Labs nodes:
-    - Mainnet: https://mainnet.api.tez.ie
-    - Hangzhounet: https://hangzhounet.api.tez.ie protocol `PtHangz2aRngywmSRGGvrcTyMbbdpWdpFKuS4uMWxg2RaH9i1qx`
-    - Ithacanet: https://ithacanet.ecadinfra.com/ protocol: `PsiThaCaT47Zboaw71QWScM8sXeMM7bbQFncK9FLqYc6EKdpjVP`
-- SmartPy public Tezos nodes
-    - Mainnet: https://mainnet.smartpy.io
-    - Hangzhounet: https://hangzhounet.smartpy.io/
-    - Ithacanet: https://ithacanet.smartpy.io/
-- Blockscale public nodes operated on behalf of the Tezos Foundation: 
-    - Mainnet: https://rpc.tzbeta.net/
-    - Next protocol/version testnet (Ithacanet): https://rpczero.tzbeta.net/
-- Teznode Public nodes operated by LetzBake!:
-    - Mainnet: https://teznode.letzbake.com
-- ~~Tezos Giga Node from Tezos Ukraine~~ Was expected cease to operations on Oct 1st 2021, but status is unkown. See
-  [Announcment](https://twitter.com/GigaNode/status/1435265400699342854)
-    - ~~Mainnet: https://mainnet-tezos.giganode.io~~
-    - Hangzhou testnet: https://testnet-tezos.giganode.io/
+| Provider         | Net         | URL                                | Header      |  
+|------------------|-------------|------------------------------------|-------|
+| ECAD Labs        | Mainnet     | https://mainnet.api.tez.ie         | [Check](https://mainnet.api.tez.ie/chains/main/blocks/head/header)      |
+| ECAD Labs        | Hangzhounet | https://hangzhounet.api.tez.ie     | [Check](https://hangzhounet.api.tez.ie/chains/main/blocks/head/header)     |
+| ECAD Labs        | Ithacanet   | https://ithacanet.ecadinfra.com    | [Check](https://ithacanet.ecadinfra.com/chains/main/blocks/head/header)      |
+| SmartPy          | Mainnet     | https://mainnet.smartpy.io         |  [Check](https://mainnet.smartpy.io/chains/main/blocks/head/header)     |
+| SmartPy          | Hangzhounet | https://hangzhounet.smartpy.io/    | [Check](https://hangzhounet.smartpy.io/chains/main/blocks/head/header)      |
+| SmartPy          | Ithacanet   | https://ithacanet.smartpy.io/      | [Check](https://ithacanet.smartpy.io/chains/main/blocks/head/header)    |
+| Tezos Foundation | Mainnet     | https://rpc.tzbeta.net/            | [Check](https://rpc.tzbeta.net/chains/main/blocks/head/header)      |
+| Tezos Foundation | Ithacanet   | https://rpczero.tzbeta.net/        | [Check](https://rpczero.tzbeta.net/chains/main/blocks/head/header)      |
+| LetzBake!        | Mainnet     | https://teznode.letzbake.com       |  [Check](https://hangzhounet.api.tez.ie/chains/main/blocks/head/header)     |
```

Comment:
wrong URLs for the Header column for the 3 last nodes of the table

Reply:
Whoops!  Fixed, thanks!